cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -196,6 +196,12 @@
     # NOTE: at this stage the /snap directory is stable as we have called
     # pivot_root already.
 
+    # Support the overmount interface which enables a custom userspace
```

Comment:
Have a look around the file, I suspect you can kill some rules with this one over-reaching rule.